src/app/modules/main/stickers/models/sticker_list.nim

```diff
@@ -0,0 +1,60 @@
+import NimQml, Tables, sequtils
+import status/chat/stickers
```

Comment:
@jrainville same as above, don't rely on anything from status-lib except methods which do status-go API calls (this and a line below).